### DISCUSSION TITLE
Hello everyone! Nice to meet you!

### DISCUSSION THREAD

**Discussion by yemikifouly** [OpenMined/courses#3] [Introduce Yourself]
Hello! 👋 

I am Yemissi Kifouly but please call me Yemi. Previously a member of the OpenMined Mentorship Team and the lead of the OpenMined Learning Team (currently on hiatus), I am now a member of the Education Team and have helped set up the support services that you will receive while [learning](https://courses.openmined.org) with us.

I am also a Senior Mentor at Udacity, a budding Product Manager, a data lover, an avid foreign languages learner and overall, an addict of anything challenging and seemingly impossible to solve or master. 😶 

On this board, I will serve primarily as a Community Manager. You can find me in the same role in the **#courses** channel of our [Slack workspace](https://slack.openmined.org).

I look forward to interacting with all of you and supporting you through your learning journey! 😀 

**Welcome to the discussion board!** 

<br>
<p align="center"><img src="https://media.giphy.com/media/l0HlHFRbmaZtBRhXG/source.gif"></p>

**Comment by aristizabal95** [comment]
Hi Yemi! I remember you from the Secure & Private AI Challenge! Glad to see you here! 😀

**Reply by elie-wanko** [reply]
Hey Yemi, nice to have you lead us!
May the code be with y'all.

**Reply by yemikifouly** [reply]
@aristizabal95 Alejandro!!! Man! So good to see you here! Still remember you sharing about how you went from music to AI and leaving me "con la boca abierta". 😃 It really is great seeing you here! Welcome! 

@elie-wanko The code shall be with us all indeed! 😃 Welcome to the discussion board!

**Comment by prafullkotecha** [comment]
Hey everyone! Looking forward to learning something new!

**Comment by bhadreshpsavani** [comment]
Hi Everyone! Looking forward to learn more!

**Comment by Azuremis** [comment]
Hi everyone, I'm excited to know more about private AI 🎉 I'm currently teaching myslef machine learning and I'm super interested to see how I can unlock the true power of data via private tech 🔥

**Comment by Foxxysentertainment** [comment]
Hello it's so nice  to be here and meet everyone

**Comment by babethling** [comment]
Hi everyone, nice to meet you all.

**Comment by prabhuSub** [comment]
Happy to be here. Thanks a lot @yemikifouly !🎉
